src/requests/walletrequests.cpp

```diff
@@ -0,0 +1,172 @@
+#include "requests/walletbalancerequest.h"
+#include "requests/walletcreateaccountrequest.h"
+#include "requests/walletrestoreaccountrequest.h"
+#include "requests/walletpreparetransferrequest.h"
+#include "requestdefines.h"
+#include "wallet_manager.h"
+
+namespace
+{
+
+const char* WEBHOOK_CALLBACK_HEADER_TAG = "Callback";
+
+std::string getCallbackString(const graft::Input& input)
+{
+    const std::vector<std::pair<std::string, std::string>>& headers = input.headers;
+
+    for (const std::pair<std::string, std::string>& header : headers)
+    {
+        if (header.first != WEBHOOK_CALLBACK_HEADER_TAG)
+            continue;
+
+        const std::string& url = header.second;
+
+        if (url.find("http") == 0) //general case - remote host exists
+            return url;
+
+        std::string full_url = "http://" + input.remote_host + ":" + url;
+
+        return full_url;
+    }
+
+    return std::string();
+}
+
+}
+
+namespace graft {
+
+Status walletCreateAccountRequestHandler
+ (const Router::vars_t& vars, 
+  const graft::Input&   input,
+  graft::Context&       context,
+  WalletManager&        wallet_manager,
+  graft::Output&        output)
+{
+    WalletCreateAccountRequestJsonRpc request;
+
+    if (!input.get(request)) {
+        return errorInvalidParams(output);
+    }
+
+    wallet_manager.createAccount(context, request.params.Password, request.params.Language, getCallbackString(input));
+
+    WalletCreateAccountResponse out;
+
+    out.Result = 0;
```

Comment:
You already have added an initialization. It is not required, but up to you.

Reply:
Fixed